app.js

```diff
@@ -60,6 +60,7 @@ function sumAndMultiply(a, b, c) {
 testSumAndMultiply();
 
 //console.log(testSumAndMultiply(4,7,5));
+
```

Comment:
Extra newlines like this scattered around your file make me nervous. Make sure that before you commit, you check what it is you're committing; in this case it's fine, because it's just an extra line, but sometimes you'll have new characters that you accidentally type that break everything, so it's a good idea to make sure you don't have unrelated nearby changes to your file in a commit.